core/kernel/sub.mk

```diff
@@ -54,6 +54,8 @@ srcs-y += pseudo_ta.c
 
 srcs-$(CFG_TRANSFER_LIST) += transfer_list.c
 
+srcs-$(CFG_SEMIHOSTING) += semihosting.c
```

Comment:
Please add `CFG_SEMIHOSTING ?=n` from the last commit (as well as the associated comments) in "core: kernel: Add semihosting functions".

Reply:
Done https://github.com/OP-TEE/optee_os/pull/6700/commits/20b8d1656665c1ab1b41dc001421ec880ec7ec9b#diff-6301a2703b709daa4d84ead1c30bf67cdaf088abed68753a8f9c740f1e58b54a